packages/big-design/src/components/Select/Select.tsx

```diff
@@ -20,6 +20,7 @@ import { SelectAction, SelectOption, SelectOptionGroup, SelectProps } from './ty
 export const Select = typedMemo(
   <T extends any>({
     action,
+    description,
```

Comment:
You shouldn't need to destructure this, since it gets passed into the `Input` via `rest`.